src/java/edu/slu/tpen/servlet/CanvasServlet.java

```diff
@@ -63,7 +67,14 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
                     resp.setHeader("Access-Control-Allow-Headers", "*");
                     resp.setHeader("Access-Control-Expose-Headers", "*"); //Headers are restricted, unless you explicitly expose them.  Darn Browsers.
                     resp.setHeader("Cache-Control", "max-age=15, must-revalidate");
-                    resp.getWriter().write(export(buildPage(f)));
+
+                    if (req.getHeader("Accept") != null && req.getHeader("Accept").contains("iiif/v3")) {
```

Comment:
This looks a whole lot like the check in ProjectServlet.java...

Not required for this PR, but it might be worth while to make a `JsonHelper.getAccetpHeader(request)` so you can use it for any servlet that has content negotation.